lib/lists/list_indexing.dart

```diff
@@ -42,6 +42,23 @@ class ListIndexing extends ConsumerWidget {
             'valid': true
           });
         });
+      } else {
+        showDialog(
+            context: context,
+            builder: (BuildContext context) {
+              return AlertDialog(
+                  title: const Text('No fields available'),
+                  content:
+                      const Text('There is no fields collection in Firestore'),
```

Comment:
No fields dictionary available. Please contact system administrator.

Reply:
No fields dictionary is available for the list ${listId}. Please contact system administrator.